adam-cli/src/test/scala/org/bdgenomics/adam/cli/Fasta2ADAMSuite.scala

```diff
@@ -30,8 +30,8 @@ class Fasta2ADAMSuite extends ADAMFunSuite {
     val contigFragments = sc.loadParquetContigFragments(convertPath)
     assert(contigFragments.rdd.count() === 26)
     val first = contigFragments.rdd.first()
-    assert(first.getContig.getContigName === "gi|224384749|gb|CM000682.1|")
-    assert(first.getDescription === "Homo sapiens chromosome 20, GRCh37 primary reference assembly")
+    assert(first.getContig.getContigName === null)
```

Comment:
Well, `gi` is the Entrez ID, and `gb` is the GenBank ID. I assure you that they are not the sequence name.